### PR TITLE
Run chrome/firefox in Travis (not sauce).

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

**Comment by rwjblue** [comment]
@krisselden / @stefanpenner - Ready for review...

File: bin/run-travis-browser-tests.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    child.stdout.on('data', function(data) {
+      console.log(data.toString());
+    });
+
+    child.stderr.on('data', function(data) {
+      console.error(data.toString());
+    });
+
+    child.on('error', function(err) {
+      reject(err);
```
Comment (stefanpenner) [inline bin/run-travis-browser-tests.js]: we may need to force kill the child here: https://nodejs.org/api/child_process.html#child_process_child_kill_signal

File: bin/run-travis-browser-tests.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  })
+  .then(function() {
+    return run('./node_modules/.bin/testem', ['launchers']);
+  })
+  .then(function() {
+    return run('./node_modules/.bin/testem', ['ci', '--file', 'testem.travis-browsers.js', '--port', '7000']);
+  })
+  .catch(function(error) {
+    console.log('error:');
+    console.log(error);
```
Comment (stefanpenner) [inline bin/run-travis-browser-tests.js]: should we also print the stack?

**Comment by stefanpenner** [comment]
some comments but largely looks great.

---

Future work: this may make a really good add-on, that our travis users could install and get great FF/chrome coverage right from there CLI apps.

**Pull request merged**